src/morkparser.cpp

```diff
@@ -276,13 +276,12 @@ void MorkParser::parseDict()
 			switch ( cur )
 			{
 			case '<':
-				if ( morkData_.mid( morkPos_ - 1, strlen( MorkDictColumnMeta ) ) 
```

Comment:
This file seems to be saved with tabs instead of spaces. Spaces are the standard for the rest of the repository. Should we convert this file to spaces as well?

Reply:
spaces please.